acra/acra-in-depth/architecture/acraserver.md

```diff
@@ -74,7 +74,7 @@ Other connections are optional – for example, you can use Redis as external ke
 
 * [AcraConnector](/acra/security-controls/transport-security/acra-connector) – optional client-side service/daemon that implements transport security and authentication for client application that doesn't support TLS 1.2+.
 
-* [AcraWebConfig](/acra/configuring-maintaining/general-configuration/acra-webconfig/) — optional lightweight HTTP web server for managing AcraServer's certain configuration options at runtime by clicking rather than updating configuration files manually.
+* [AcraWebConfig](/acra/configuring-maintaining/general-configuration/acra-webconfig/) (deprecated after 0.90.0) — optional lightweight HTTP web server for managing AcraServer's certain configuration options at runtime by clicking rather than updating configuration files manually.
```

Comment:
```suggestion
* [AcraWebConfig](/acra/configuring-maintaining/general-configuration/acra-webconfig/) (deprecated since 0.91.0) — optional lightweight HTTP web server for managing AcraServer's certain configuration options at runtime by clicking rather than updating configuration files manually.
```